app/schema/user.py

```diff
@@ -81,6 +79,7 @@ class UsersSearchOut(BaseModel):
 
 
 class UserProfileOut(User):
+    auth_account: list[AuthAccount] = []
```

Comment:
Info about auth accounts should NEVER leave backend and it should NEVER be sent to a client